plugins/slick.autotooltips.js

```diff
@@ -56,7 +56,9 @@
             if (options.maxToolTipLength && text.length > options.maxToolTipLength) {
               text = text.substr(0, options.maxToolTipLength - 3) + "...";
             }
-          } else {
+          } else if ($node.innerHeight() < $node[0].scrollHeight) {
```

Comment:
why not merge this `else if` into the first `if`? I think this would be better and it would also include the line 56-58 for longer text tooltip

Reply:
That is a good suggestion!  I've updated the code.  Thanks!